<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QscOptionDialog</class>
 <widget class="QDialog" name="QscOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/icons/pascal64.png</normaloff>:/icons/icons/pascal64.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>451</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
   </property>
  </widget>
  <widget class="QListWidget" name="m_categoriesLW">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>161</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <property name="selectionRectVisible">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Enviropment</string>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/category_core.png</normaloff>:/icons/icons/category_core.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Text Editor</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/category_texteditor.png</normaloff>:/icons/icons/category_texteditor.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pascal</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/pascal64.png</normaloff>:/icons/icons/pascal64.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Debug</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/category_debug.png</normaloff>:/icons/icons/category_debug.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Help</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/category_help.png</normaloff>:/icons/icons/category_help.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>521</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="m_envPage">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>251</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="m_generalTab">
       <attribute name="title">
        <string>General</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="m_keyboardTab">
       <attribute name="title">
        <string>Keyboard</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="m_editorPage">
     <widget class="QTabWidget" name="m_editorTabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>501</width>
        <height>261</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="m_fontTab">
       <attribute name="title">
        <string>Fonts &amp;&amp; Colors</string>
       </attribute>
       <widget class="QLabel" name="m_fontPreviewLabel">
        <property name="geometry">
         <rect>
          <x>86</x>
          <y>100</y>
          <width>291</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>This is a sample font preview</string>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>480</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Family:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="m_fontComboBox">
           <property name="currentFont">
            <font>
             <family>DejaVu Sans Mono</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_fontSizeCB">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Zoom:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_fontZoomSB">
           <property name="maximum">
            <number>300</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="m_behaviorTab">
       <attribute name="title">
        <string>Behavior</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="m_displayTab">
       <attribute name="title">
        <string>Display</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="m_completionTab">
       <attribute name="title">
        <string>Completion</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="m_pascalPage"/>
    <widget class="QWidget" name="m_debugPage"/>
    <widget class="QWidget" name="m_helpPage"/>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>167</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="m_searchLE">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string comment="Filter"/>
      </property>
      <property name="placeholderText">
       <string>Filter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_titleLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QscOptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QscOptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
